server/src/test/java/org/opensearch/index/remote/RemoteSegmentTransferTrackerTests.java

```diff
@@ -154,13 +154,13 @@ public void testComputeTimeLagOnUpdate() throws InterruptedException {
         transferTracker.updateLatestLocalFileNameLengthMap(List.of("test"), k -> 1L);
         // Sleep for 100ms and then the lag should be within 100ms +/- 20ms
         Thread.sleep(100);
-        assertTrue(Math.abs(transferTracker.getTimeMsLag() - 100) <= 20);
+        assertTrue(transferTracker.getTimeMsLag() >= 100);
```

Comment:
We should add an error message to maybe log the value of the lag if this assertion fails. helps with debugging in future